stable/envoy/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Envoy is an open source edge and service proxy, designed for cloud-native applications.
 name: envoy
-version: 1.8.0
```

Comment:
@khorshuheng Hey! Thanks for the fix. Could you rebase onto the latest `master` and change the version number here to `1.8.2`?

Reply:
I have rebased and update the version number.

Reply:
@khorshuheng LGTMed. Thanks for your effort 👍